### ISSUE TITLE
RSS Reader for weeklyosm

### ISSUE THREAD

**Issue by westnordost** [streetcomplete/StreetComplete#1796]
Using the notifications button functionality, make the app into an RSS reader for weeklyosm, optionally only after the user unlocked the achievement that contains the link to it and optionally with an option to unsubscribe from the weekly updates.

The reader could function in a very simple way: Just checking if there is a new issue and notifying the user of it, opening the browser if he wants to read it. Similarly to the "you have mail in your OSM inbox" notification.

An animation for when the notification button is clicked could look like this: https://www.youtube.com/watch?v=saxwDCYVksQ

**Comment by westnordost** [comment]
I submitted an [application for a microgrant at the OSMF](https://wiki.openstreetmap.org/wiki/Microgrants/Microgrants_2020/Proposal/StreetComplete_as_an_entry_point_to_OpenStreetMap) to develop this feature.

**Comment by rugk** [comment]
Saw that you reopened this: My IMHO is it's a little nice "gadget"/"nice to have" tool/feature/achievement (if we talk in gamification terms), so yeah could be worth it. Technically, I am unsure how complex RSS parsing here is or how good existing Kotlin(?) libraries for this task are, but generally from a users perspective such a thing is marketing people would call "customer retention" aka keeping your user (in this case) in the OSM world and offer even more paths for more things to do…

**Comment by westnordost** [comment]
https://github.com/user-attachments/assets/e4c06218-0eff-4445-ba4d-7d384c4bd74b

Working on UI
